### ISSUE TITLE
Integers as enum constraints not supported by sqlalchemy in sql plugin

### ISSUE THREAD

**Issue by rbngz** [frictionlessdata/frictionless-py#776]
# Overview

According to the frictionless [data specs](https://specs.frictionlessdata.io/table-schema/#constraints) a constraint of the type 'enum' applies to all data types. However, when having integers as enum constraints, the SQL plugin will call sqlalchemy.Enum() which does not take integers as values.

```
elif const == "enum":
      enum_name = "%s_%s_enum" % (sql_name, field.name)
      column_type = sa.Enum(*value, name=enum_name)
```
 This will lead to a TypeError within the sqlalchemy code. 

This can be avoided by creating a [custom enum type](https://docs.sqlalchemy.org/en/14/core/custom_types.html#augmenting-existing-types) that is supported by sqlalchemy.

See also this [example](https://stackoverflow.com/questions/33612625/how-to-model-enums-backed-by-integers-with-sqlachemy).

---

Please preserve this line to notify @roll (lead of this repository)

**Comment by roll** [comment]
Thanks! I'll fix it

**Comment by roll** [comment]
Hi @rbngz,

I've quick-fixed the error in #779.

Would you be interested in creating a PR for #778?
